core/base/topologicalCompression/TopologicalCompression.h

```diff
@@ -15,7 +15,7 @@
 
 // base code includes
 #include <FTMTreePP.h>
-#include <TopologicalSimplification.h>
+#include <LegacyTopologicalSimplification.h>
```

Comment:
TopologicalSimplification

Reply:
use the legacy instead (based on discussions) because the compression is using specific features.